experimental/bits/simd.h

```diff
@@ -3559,8 +3565,7 @@ split(const simd_mask<typename _V::simd_type::value_type, _Ap>& __x)
 
 // }}}
 // split<_Sizes...>(simd) {{{
-template <size_t... _Sizes, typename _Tp, typename _Ap,
-	  typename = enable_if_t<((_Sizes + ...) == simd<_Tp, _Ap>::size())>>
+template <size_t... _Sizes, typename _Tp, typename _Ap, typename>
```

Comment:
removing the SFINAE condition breaks the spec

Reply:
SFINAE is there in declaration. Here it is definition. clang says it is redefinition of default argument

Reply:
Declaration is at line 3314

Reply:
OK thanks. I'll take a look